src/utils/typography.js

```diff
@@ -1,8 +1,9 @@
-import Typography from 'typography';
-import Wordpress2016 from 'typography-theme-wordpress-2016';
 import '../fonts/fonts-shared.css';
 import './global.css';
 
+import Typography from 'typography';
+import Wordpress2016 from 'typography-theme-wordpress-2016';
+
```

Comment:
Woops, sorry about the import re-ordering, my IDE does that. It shouldn't have any side effects (I noticed none), but I can revert that if you wish by disabling the relevant ext in my local workspace.